packages/website/docs/MultilineBox.md

```diff
@@ -0,0 +1,45 @@
+# Best Practices
+The Multiline box allows users to **add the same sequence of inputs multiple times**.  
+Each sequence appears on a different line, making it easier to consult.  
+
+Do not use the multiline box to regroup inputs that have no connection to one another.   
+Multiline works best when the sequence of inputs is identical on each line.  
+
+Multiline box can combine multiple inputs together to form a more complex input.  
+They can also combine text elements when an input or sequence requires more explanation.  
+
+Only use the drag and drop function if the order of each sequence is important.   
+When doing so, the list must always be presented to the user in the order they have set.
+
+# Labeling
+Keep titles short, preferably **under three words**.  
+
+If the list of inputs is very simple, provide a descriptive title without action verbs.  
+For example, write "Email list" rather than "Provide a list of emails”.
+
+If the sequence of inputs is more complex, consider using a title with a trigger verb to clearly explain that the outcome will apply or consider the entire list.  
+For example, write “Show content that matches" instead of "Filter rules”. 
+
+When using a dialog approach, always consider other inputs in the form so that it is cohesive with other inputs.
+
+# Help Text and Instructions
+Avoid the need for help text or instruction for each individual input that appears in a multiline line. 
+
+Instead, provide short instructions, above the lines, that will briefly explain the type of inputs to be provided in the input sequence.
+
+# Feedback and Validation
+Make sure that a first sequence is **completed and valid** before allowing the addition of a new line. 
+
+If the multiline is optional, do not show an empty first line. Users should be required to add it manually if they need it. 
```

Comment:
This is not a behaviour the component currently implements AFAIK

Reply:
The component indeed renders a new "empty" box by default when it mounts, it is in fact problematic for my current work with the `queryExpressionBuilder` and I though about adding some `doNotAddAnEmptyBoxByDefaultBecauseIDontWantOne` prop down the road.

Reply:
```suggestion
If the multiline box is optional, don't show an empty first line. Users should add it manually if they need it. 
```